database/src/main/java/entities/Book.java

```diff
@@ -17,17 +24,29 @@
 
     private String authors;
 
+    @Column(length = 10000)
     private String description;
 
-    // add library list (entity), tag list (entity), category id (entity), discount type (enum)
+    private String discount;
+
+    private String price;
+
+    private Timestamp timestamp;
```

Comment:
Careful with Timestamp. Remember to annotate it properly, depending on time type you wanna use on DB side.